client/src/components/footer/footer.js

```diff
@@ -185,17 +185,19 @@ class Footer extends Component {
             style={{ display: display ? "block" : "none" }}
           >
             <button
-              className="btn tw-bg-labLightGray btn-xl text-uppercase back "
+              //className="btn tw-bg-labLightGray btn-xl text-uppercase back "
```

Comment:
You can delete unused code, we can always look back at older commits if we need to get this code back